app/src/src/utils/routes.js

```diff
@@ -15,7 +15,11 @@ export const routes = Object.freeze({
   menuHome: 'MenuHome',
   profile: 'Profile',
   editProfile: 'EditProfile',
-  savedItems: 'SavedItems',
+  savedHome: 'SavedHome',
+  posts: 'Posts',
+  events: 'Events',
+  ads: 'Ads',
+  listings: 'listings',
```

Comment:
We should add the prefix 'saved' to these routes just so that we don't have confusion in the future (e.g. savedPosts)

Reply:
done